templates/components/content_blocks/repository_summary.html

```diff
@@ -49,13 +49,13 @@ <h4 class="govuk-heading-s">Activity</h4>
               </dd>
             </div>
             {% endif %}
-            {% if repo.recentCommitDaysAgo %}
+            {% if repo.recentCommitDaysAgo is not none %}
```

Comment:
1. This seems almost identical logic?
2. Do you mean `None`, capital N?

Reply:
The reason for this is the case where the value is 0 (committed today) which is falsy. 
For some reason None is "none" in Jinja syntax

Reply:
Huh. OK!